core/common/src/main/java/alluxio/master/file/MasterMetadataSyncTraverseType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file;
+
+/**
+ * The pending Path in the Inode SyncStream traversal order.
```

Comment:
Actually, do you think it makes sense to make it a user side configuration? If we use a master-side configuration then all the sync operations will be either BFS or DFS. But imagine we add a config like `alluxio.user.file.metadata.sync.traversal.order=DFS` you can make certain jobs use DFS, while all other jobs are still BFS.

In other words, if the majority of sync are small, while you have some unavoidable large sync operations on super large dirs, then you might want to set the default `alluxio.user.file.metadata.sync.traversal.order=BFS` and specify DFS on those large syncs.

WDYT? @adol001 @elega @tcrain